addons/sale_stock/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,45 @@
+---Fields in module 'sale_stock'---
+sale_stock   / procurement.group        / sale_id (many2one)            : NEW relation: sale.order
+# Nothing to do
+
+sale_stock   / sale.order               / picking_ids (many2many)       : not a function anymore
+sale_stock   / sale.order               / picking_ids (many2many)       : type is now 'one2many' ('many2many')
+# Nothing to do
+
+sale_stock   / sale.order               / procurement_group_id (many2one): previously in module sale
+# Done, rename references on sale pre-migration script
+
+sale_stock   / sale.order.line          / move_ids (one2many)           : NEW relation: stock.move
+sale_stock   / stock.move               / sale_line_id (many2one)       : NEW relation: sale.order.line
```

Comment:
This field should be filled also or the work flows will break. You have to get it from old procurement orders.

Reply:
@pedrobaeza I wonder if something similar would be needed for the field move_ids